src/MonoTorrent.BEncoding/MonoTorrent.BEncoding.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Condition="$(TargetFramework) == 'netstandard2.0'" Include="System.Memory" Version="4.5.0" />
+    <PackageReference Condition="$(TargetFramework) == 'netstandard2.0'" Include="System.Memory" Version="4.5.1" />
```

Comment:
we need 4.5.1 because of the addition of `StandardSocketsHttpHandler`. No worries!